### PR TITLE
Fix links of inherited properties, do not generate pages for them

### PR DESCRIPTION
**Pull request by vmishenev**

### CODE REVIEW

File: plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt [hunk 1]
```diff
@@ -51,7 +51,7 @@ open class DefaultPageCreator(
             e.nameAfterClash(), contentForEnumEntry(e), setOf(e.dri), e,
             e.classlikes.renameClashingDocumentable().map(::pageForClasslike) +
                     e.filteredFunctions.renameClashingDocumentable().map(::pageForFunction) +
-                    e.properties.renameClashingDocumentable().mapNotNull(::pageForProperty)
+                    e.filteredProperties.renameClashingDocumentable().mapNotNull(::pageForProperty)
```
Comment (MarcinAman) [inline plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt:54]: It think this filters out the entries even when we have `suppressObviousMembers = true` and / or `suppressInheritedMembers = true`, shouldn't this be filtered out in `InheritedEntriesDocumentableFilterTransformer`?

File: plugins/base/src/test/kotlin/model/PropertyTest.kt [hunk 2]
```diff
@@ -138,6 +138,29 @@ class PropertyTest : AbstractModelTest("/src/main/kotlin/property/Test.kt", "pro
         }
     }
 
+    @Test
```
Comment (MarcinAman) [inline plugins/base/src/test/kotlin/model/PropertyTest.kt:141]: Could you also update the previous test (`propertyOverride`) to include assertion about DRI?

**Review by MarcinAman — CHANGES_REQUESTED** [review]

File: plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt (hunk 1 above)
Comment (vmishenev) [reply plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt:54]: We don't have `isObviousMembers` for properties. 
It should be filtered out in `InheritedEntriesDocumentableFilterTransformer` as well if  `suppressInheritedMembers` is set (by default it's `false`). 
This filter only allows to not generate the pages for inherited properties, but the properties  are kept.

Comment (MarcinAman) [reply plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt:54]: >We don't have isObviousMembers for properties.
This is not good overall but ok, for this PR

**Review by MarcinAman — APPROVED** [review]

**Pull request merged**